modules/contact/src/ContactMaster.C

```diff
@@ -407,24 +427,21 @@ contactModel(const std::string & the_name)
   else if ("experimental" == name)
   {
     model = CM_FRICTIONLESS;
```

Comment:
We probably deprecated this about 4 years ago, so it's time to just delete this.